server/lib/models/device.go

```diff
@@ -4,7 +4,7 @@ import (
 	//"log"
 	"time"
 
-	proto "../../proto"
+	proto "github.com/umegaya/channer/server/proto"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/proto (can't find import: github.com/umegaya/channer/server/proto)